packages/material-ui-unstyled/src/FormControlUnstyled/FormControlUnstyled.test.tsx

```diff
@@ -0,0 +1,200 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import { spy } from 'sinon';
+import {
+  createMount,
+  createClientRender,
+  describeConformanceUnstyled,
+  fireEvent,
+} from 'test/utils';
+import FormControlUnstyled, {
+  formControlUnstyledClasses,
+  useFormControlUnstyled,
+} from '@material-ui/unstyled/FormControlUnstyled';
+
+function TestComponent() {
+  const context = useFormControlUnstyled();
+
+  const inputProps = {
+    'data-filled': context?.filled,
+    'data-focused': context?.focused,
+    disabled: context?.disabled,
+    onChange: context?.onChange,
+    required: context?.required,
+    value: context?.value as any,
```

Comment:
Why the type cast here?

Reply:
`context.value` has `unknown` type. I could set its type to `string | number | readonly string[]` to match the native input's props, but this would limit the usage of FormControl with other types of inputs (such as DatePicker).
Another option would be to just use `any` instead of `unknown`.

Reply:
Then let's cast it to `string` so that we understand what we're expecting here.